frontend/src/app/forms/forms.page.html

```diff
@@ -0,0 +1,55 @@
+<ion-header class="ion-no-border">
+  <ion-toolbar>
+    <ion-grid>
+      <ion-row>
+        <ion-col class="font30">
+          <ion-icon name="close" (click)="onClose()"></ion-icon>
+        </ion-col>
+        <ion-col class="ion-text-center ion-float-center progresstext">
+          Step {{currentMenu.id}}/ {{totalSteps}}
+        </ion-col>
+        <ion-col>
+          <ion-menu-button
+            class="ion-float-right font30"
+            autoHide="true"
+            menuClose
+            id="main"
+          >
+            <ion-icon name="list"></ion-icon>
+          </ion-menu-button>
+        </ion-col>
+      </ion-row>
+    </ion-grid>
```

Comment:
Yes it was a bit messy. And does not suit all device. The main purpose having row column is to form a grid with three columns so that the elements can be positioned well. Will check for the other possibilities too.

Reply:
I believe my suggestion (using the "slot" attribute) should do the trick. The Ionic docs should provide some more examples.